### PR TITLE
fix: Contour multiplexing translate had the wrong arguments, add UL interpolation

### PR DESCRIPTION
**Pull request by longjon929**
This is mostly to add the xs UL interpolation and fix a weird corner case where shapes have a line hanging off of them:
image
which breaks part of the region merging.  I've also added an option to draw the best expected regions over the entire plane.

While testing, I found there was a problem with the translate() call that I fixed.

### CODE REVIEW

File: scripts/multiplexContours.py [hunk 1]
```diff
@@ -619,6 +714,58 @@ def plotBestRegion(bestRegions, expCurve):
...
+                            break
+                    break
+
+    x_out = [o[0] for o in outpoints]
+    y_out = [o[1] for o in outpoints]
+    z_out = [o[2] for o in outpoints]
+
+    #print(x_out)
+    #print(y_out)
+    #print(z_out)
```
Comment (kratsg) [inline scripts/multiplexContours.py:766]: these should be removed?

File: scripts/multiplexContours.py [hunk 2]
```diff
@@ -584,6 +669,8 @@ def doPolygonsTouch(poly1, poly2):
 
 def plotBestRegion(bestRegions, expCurve):
 
+    fout = ROOT.TFile("usedExpectedContours.root","recreate")
```
Comment (kratsg) [inline scripts/multiplexContours.py:672]: maybe just wrap this in a 

```
with ROOT.TFile("....") as fout:
  ...
```

then you drop the `fout.Close()`.

**Review by kratsg — CHANGES_REQUESTED** [review]

File: scripts/multiplexContours.py (hunk 2 above)
Comment (longjon929) [reply scripts/multiplexContours.py:672]: Sure

File: scripts/multiplexContours.py (hunk 1 above)
Comment (longjon929) [reply scripts/multiplexContours.py]: Sure

File: scripts/multiplexContours.py (hunk 2 above)
Comment (longjon929) [reply scripts/multiplexContours.py:672]: Doesn't seem to work with ROOT files.

Comment (kratsg) [reply scripts/multiplexContours.py:672]: you're right, wow. i am thinking of uproot 🤣

**Review by kratsg — APPROVED** [review]

**Pull request merged**